packages/data/src/defs/entrances.yml

```diff
@@ -57,7 +57,7 @@ OOT_KAKARIKO_FROM_DEATH_MOUNTAIN:               { game: oot, id: 0x191, type: re
 OOT_FIELD_FROM_MARKET_ENTRANCE:                 { game: oot, id: 0x1fd, type: region-exit, maps: [OOT_MARKET_ENTRANCE, OOT_HYRULE_FIELD], areas: ["OOT Market Entryway", "OOT Hyrule Field"], reverse: OOT_MARKET_ENTRANCE_FROM_FIELD }
 OOT_LOST_WOODS_FROM_ZORA_RIVER:                 { game: oot, id: 0x4da, type: region-exit, maps: [OOT_ZORA_RIVER, OOT_LOST_WOODS], areas: ["OOT Zora River", "OOT Lost Woods"], reverse: OOT_ZORA_RIVER_FROM_LOST_WOODS }
 OOT_LOST_WOODS_FROM_GORON_CITY:                 { game: oot, id: 0x4d6, type: region-exit, maps: [OOT_GORON_CITY, OOT_LOST_WOODS], areas: ["OOT Goron City Shortcut", "OOT Lost Woods"], reverse: OOT_GORON_CITY_FROM_LOST_WOODS }
-OOT_ZORA_DOMAIN_FROM_LAKE_HYLIA:                { game: oot, id: 0x328, type: region-exit, maps: [OOT_LAKE_HYLIA, OOT_ZORA_DOMAIN], areas: ["OOT Lake Hylia", "OOT Zora Domain Shortcut"], reverse: OOT_LAKE_HYLIA_FROM_ZORA_DOMAIN }
+OOT_ZORA_DOMAIN_FROM_LAKE_HYLIA:                { game: oot, id: 0x328, type: region-exit, maps: [OOT_LAKE_HYLIA, OOT_ZORA_DOMAIN], areas: ["OOT Lake Hylia Near Shortcut", "OOT Zora Domain Shortcut"], reverse: OOT_LAKE_HYLIA_FROM_ZORA_DOMAIN }
```

Comment:
Zora Domain side should put you in Zora Domain Underwater, since you require iron boots to get back into the entrance if you don't or can't surface. Lake Hylia side should be coming from Lake Hylia, since all methods to get in from there guarantee passage through and any methods coming out also give access to the lake hylia region.